examples/functions-as-entry-points/functions_as_entry_points.out

```diff
@@ -0,0 +1,34 @@
+# At the command line, navigate to the directory that contains the `.bal` file and
+# and use the `ballerina run` command with the `--printreturn` option set to print
+# the return value to the standard out stream.
+
+# use the `ballerina run` command to invoke the `main` function with `add` as the
+# string argument for `op`, and `4` and `8` as the integer arguments on which the
+# operation is to be performed.
+$ ballerina run --printreturn functions_as_entry_points.bal add 4 8
+Operation: Addition
+12
+
+# use the `ballerina run` command specifying `add` as the function to invoke and `14`
+# and `8` as the integer arguments to add.
```

Comment:
Minor change:
"# Use the `ballerina run` command with `add` as the function to invoke, and specify`14`
# and `8` as the integer arguments to add."

Reply:
Might have to do this change in other places also if applicable.